### PR TITLE
Update CHANGELOG and README

### PR DESCRIPTION
**Pull request by onimsha**
Signed-off-by: Alex Co <alex.tuan@mindvalley.com>

## Summary

Ticket:

### Changed  

* Update README.  
* Update CHANGELOG.

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/67#" title="Author self-approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md
Comment (jk-gan) [inline README.md:59]: I believe the whole section is not needed as the `wukong` should be able to handle the config file breaking changes. 
Or we could just say:

# IMPORTANT !!

Since the Wukong CLI is not stable yet (not v1.0.0), users must always use the latest version. The latest version is **0.0.5-beta2**. 
If you are using Wukong version < **0.0.5-beta2`**, please follow the below instructions:

```bash
# Check the current version of Wukong CLI.  
$ wukong --version
# If your version is < 0.0.5-beta2.  
$ brew update  
$ brew upgrade wukong  
```

Comment (onimsha) [reply README.md]: good idea. I updated the README with your recommendation .

**Review by jk-gan — APPROVED** [review]
LGTM.

I wonder if we want to make the changelog more interesting, like show the major new features in this version with some screenshots.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/67#pullrequestreview-1308667868" title="Approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/67#" title="Author self-approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
